src/components/HomeBody.tsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from "react";
+import { getGithubFacebookInfo, getGithubFacebookList } from "../services/githubApi";
+
+export default function HomeBody(){
+  
+  type DataType = { id :string, 
+    user :{avatar_url :string, login :string}, 
+    number :number, 
+    body :string,
+    title :string, 
+    comments :number, 
+    labels :Array<string>}
+
```

Comment:
[imo]
かっこ`{}`とフィールドの型(idはstringとか)とを別の行にしたいです。
```suggestion
  type DataType = { 
    id :string, 
    user :{avatar_url :string, login :string}, 
    number :number, 
    body :string,
    title :string, 
    comments :number, 
    labels :Array<string>
}
```